readme.md

```diff
@@ -1,19 +1,16 @@
 # aragonOS <img align="right" src="https://raw.githubusercontent.com/aragon/design/master/readme-logo.png" height="80px" /> [![Travis branch](https://img.shields.io/travis/aragon/aragonOS/master.svg?style=for-the-badge)](https://travis-ci.org/aragon/aragonOS) [![Coveralls branch](https://img.shields.io/coveralls/aragon/aragonOS/master.svg?style=for-the-badge)](https://coveralls.io/github/aragon/aragonOS?branch=master) [![npm](https://img.shields.io/npm/v/@aragon/os.svg?style=for-the-badge)](https://www.npmjs.com/package/@aragon/os)
 
-This repo contains Aragon's reference implementation for [aragonOS](https://wiki.aragon.one/dev/aragonOS_index/).
+This repo contains Aragon's reference implementation for [aragonOS](https://hack.aragon.org/docs/aragonos-intro.html).
 
-#### 🚨 Everything in this repo is highly experimental software.
-It is not secure to use any of this code in production (mainnet) until proper security audits have been conducted. It can result in irreversible loss of funds.
-
-#### 🦋 We are using [Status Open Bounty](http://openbounty.status.im/app) to reward open source contributions with Aragon Tokens (ANT)
-All issues tagged with **[bounty](https://github.com/aragon/aragonOS/labels/bounty)** are eligible for a bounty on a successfully merged Pull Request that solves the issue. Even if the bounty says 0 ANT, if it has the **bounty** label, it will always be higher than 0 ANT (until we automate it, we may take a bit to fund the bounties manually).
+#### 🚨 Security review status: bug bounty
+aragonOS 4 has undergone two independent professional security reviews, and the issues raised have been resolved. However there is a [bug bounty program](https://wiki.aragon.org/dev/bug_bounty/) for rewarding hackers who find security vulnerabilities. There is a bounty pool of $250,000 USD, you can find more information [here](https://wiki.aragon.org/dev/bug_bounty/).
 
 #### 👋 We are tagging tasks that are [good first issues](https://github.com/aragon/aragonOS/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) so you can get started contributing to Aragon Core.
 Don't be shy to contribute even the smallest tweak. Everyone will be especially nice and helpful to beginners to help you get started!
 
 ## Documentation
 
-Visit the [Aragon wiki](https://wiki.aragon.one/dev/aragonOS_index/) for in depth documentation on the [architecture](https://wiki.aragon.one/documentation/aragonOS/) and different parts of the system.
+Visit the [Aragon Developer Portal](https://hack.aragon.org/docs/aragonos-intro.html) for in depth documentation on the [architecture](https://hack.aragon.org/docs/aragonos-ref.html) and different parts of the system.
 
 ## Installing aragonOS

```

Comment:
While we're at it, I think we should change the heading of this too, to be something like "Get started contributing" under the "Contributing" section at the bottom.

Installing `aragonOS` is literally `npm install`.